scripts/nsls2-tag-build.py

```diff
@@ -0,0 +1,53 @@
+import os
+import datetime
+import docker
+from pathlib import Path
+
+
+def run_container(*, pkg_name, pythons=('3.5', '3.6', '3.7')):
```

Comment:
some document here? Maybe docker_image as input?

Reply:
Yeah, I'll document the args. Good idea about the docker image.